generator/app/models/RubyClientGenerator.scala

```diff
@@ -911,7 +911,8 @@ ${headers.rubyModuleConstants.indent(2)}
     Primitives(ptName).getOrElse {
       sys.error(s"Unknown primitive type[$ptName]")
     } match {
-      case Primitives.Integer | Primitives.Double | Primitives.Long | Primitives.Uuid | Primitives.Decimal | Primitives.Boolean => varName
+      case Primitives.Decimal => s"${varName}.to_f"
```

Comment:
I actually think we might want to consider turning this into a string to avoid precision issues in serialization / deserialization. WDYT?